src/views/List.jsx

```diff
@@ -39,9 +23,15 @@ export function List({ data, listPath }) {
 		item.name.includes(search.toLowerCase()),
 	);
 
+	useEffect(() => {
+		setTimeout(() => {
+			setLoadingStatus(false);
+		}, 2000);
+	}, []);
+
```

Comment:
I like the loader! Much better user experience :) I do notice though that when navigating between certain lists it's buggy (when going from a list that doesn't have any items on it to a list that does, I still see a flash of the "add first item" button, then loading, then the list). Maybe something to do with the conditional rendering below?

Reply:
I also noticed that the `<p>` tag that holds "hello from ... page" is also flashing and doesn't always correctly correspond to the correct state of the page